### DISCUSSION TITLE
how to redirect logs?

### DISCUSSION THREAD

**Discussion by mvadu** [cortexproject/cortex#4757] [Q&A] [answered]
hello, we are running Cortex in microservice mode in K8 (AKS). We are trying to setup Cortex pods to have grafana promtail agent as a sidecar and have it scrape cortex logs. We could not find any way to control which path Cortex writes its logs? at the moment the binray seems to be logging only to stdout, and thus only kubelet log pipeline can see those logs, is that true? Is there any way for us to make the binary write to pod's local volume (e.g. /var/log/cortex) so that we can configure promtail to tail the logs?

**Comment by friedrichg** [comment, marked as answer]
yeah, stdout is the standard. I have used promtail in the past, you can use promtail as a daemonset and read docker logs. But this is not the right place to ask promtail questions.
